src/server/game/Entities/Unit/StatSystem.cpp

```diff
@@ -346,7 +344,7 @@ void Player::UpdateAttackPowerAndDamage(bool ranged)
             val2 = strengthValue + agilityValue;
         }
         else
-            val2 = (level + std::max(GetStat(STAT_AGILITY), 0.0f)) * entry->RangedAttackPowerPerAgility;
+            val2 = (std::max(GetStat(STAT_AGILITY), 0.0f)) * entry->RangedAttackPowerPerAgility;
     }
```

Comment:
This is a fix which is not directly belonging here, but i found it while i worked on pets stats